.github/workflows/performance-test.yml

```
@@ -0,0 +1,516 @@
+    - name: Restore PR image cache
+      id: image_cache_pr
+      uses: actions/cache@v4
+      with:
+        path: |
+          ${{ env.CI_IMAGE_CACHE }}
+        key: ${{ github.run_id }}-image-cache-pr
+
+    - name: Check if PR image build is needed
+      id: is_pr_image_build_needed
+      continue-on-error: true
+      run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:2:27: Double quote to prevent globbing and word splitting

(shellcheck)